src/app/utils/plug-descriptions.ts

```diff
@@ -204,6 +205,14 @@ function getPerkDescriptions(
       usedStrings.add(notif);
     }
   }
+  function addSpecialDescriptions() {
+    if (plug.displayProperties.name.includes('Harmonic')) {
+      results.push({
+        perkHash: -usedStrings.size,
+        description: t('Mods.HarmonicModDescription').toString(),
```

Comment:
```suggestion
        description: t('Mods.HarmonicModDescription'),
```

Reply:
removing .toString() results in an error in DIM console seems to still work without it but I'm not sure if this is a concern or not

![image](https://github.com/DestinyItemManager/DIM/assets/98132549/d24d2bad-40d5-491f-b0d4-2ce38f0cd7e4)

Reply:
You're importing the wrong `t` -- try `import { t } from 'app/i18next-t';` in the imports instead of `import { t } from 'i18next';`.